src/cljs/main/broadfcui/components/modals.cljs

```diff
@@ -93,7 +94,8 @@
 
 (defn render-message [{:keys [header text on-confirm on-dismiss]}]
   [OKCancelForm
-   {:header (or header "Confirm")
+   {:data-test-id "push-message"
```

Comment:
same

Reply:
@MattPutnam, I wasn't too fond of them either. I was just recovering what had been there. Would `error-modal` and `message-modal` work for you? If not, please suggest something better.